index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="ja">
+
+<head>
+    <meta charset="UTF-8">
+    <title>HTML CSSカリキュラム</title>
+    <link rel="stylesheet" href="style.css">
+
+</head>
+
+<body>
+    <main>
+        <div class="sender-column">
+
+            <p>QUELCODE大学</p>
+            <p>科学部</p>
+            <p>横浜海斗博士</p>
+            <p>Tel: 123-456-7890</p>
+            <p>Email: kaito@quelcode.com</p>
+            <p>2020年1月31日</p>
+
+        </div>
+        <p>千葉大志様</p>
+        <p> 東京都渋谷区</p>
+        <p>恵比寿4-20-4</p>
+        <p>恵比寿ガーデンプレイス</p>
+        <section>
+            <h1>Re:QUELCODE大学の科学部参加の件</h1>
+            <p>千葉様</p>
+            <p>来年の博士号の一部として勉強するために、QUELCODE大学の科学部へ参加申し込みありがとうございます。
+                <br> 次のセクションで、質問に1つずつお答えします。</p>
```

Comment:
```suggestion
            <p>
              来年の博士号の一部として勉強するために、QUELCODE大学の科学部へ参加申し込みありがとうございます。<br>
              次のセクションで、質問に1つずつお答えします。
            </p>
```
と複数行の場合はこの書き方のほうが可読性高いです